shell/conf/benchmarks_defaults.conf

```diff
@@ -136,10 +125,10 @@ fi
 [ ! "$BB_PARALLEL_STREAMS" ] && BB_PARALLEL_STREAMS="2" #Number of paralel streams for the thoughput test
 [ ! "$BB_MODE" ] && BB_MODE="sequential" # Available options are: squential(default), parallel
 if [ ! "$BB_SCALE_FACTORS" ] ; then
-  if [ "$BENCH_SCALE_FACTOR" == 0 ] ; then
-    BB_SCALE_FACTORS=$((BB_SCALE_FACTORS + 1))
+  if [ "$BENCH_DATA_SIZE_GB" -eq 0 ] ; then
+    BB_SCALE_FACTORS=1
   else
-    BB_SCALE_FACTORS=$BENCH_SCALE_FACTOR
+    BB_SCALE_FACTORS="$BENCH_DATA_SIZE_GB"
   fi
```

Comment:
I thought the whole idea was to remove these "scale factor" variables and use our new "data size" variables instead. If not, we're left with more complexity, not less.

Reply:
BigBench has an specific functionality: you can run concurrent bench with different data sizes. We need this variable to do so. If the user does not set BB_SCALE_FACTORS it will default to BENCH_DATA_SIZE, otherwise you can set multiple scale factors (in GB).

Reply:
Ah, sorry, I got confused by the variable names that use the singular "factor" and plural "factors". So the variable we are removing is `BENCH_SCALE_FACTOR` in this code 👍